database/redis/database.go

```diff
@@ -91,23 +92,47 @@ func NewTestDatabaseWithIncorrectConfig(logger moira.Logger) *DbConnector {
 	return NewDatabase(logger, Config{Addrs: []string{"0.0.0.0:0000"}}, testSource)
 }
 
-func (connector *DbConnector) manageSubscriptions(tomb *tomb.Tomb, channel string) (<-chan []byte, error) {
-	c := (*connector.client).Subscribe(connector.context, channel)
-	err := c.Ping(connector.context)
+func (connector *DbConnector) makePubSubConnection(channels []string) (*redis.PubSub, error) {
+	psc := (*connector.client).Subscribe(connector.context, channels...)
```

Comment:
Personally I vote for readable variable names, like `pubSubChannel` in this case and avoid using one character variable names

Reply:
I totally agree with the idea of giving more readable names to variables, but if I do it now, the diff of the epic branch will increase, and it is already huge.